accepted/2021/utf8/README.md

```diff
@@ -0,0 +1,21 @@
+# Improve UTF-8 support
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+UTF-8 is currently the most popular encoding on the web with [more than 95% of all websites using it](https://w3techs.com/technologies/cross/character_encoding/ranking). Additionally it's used as a default encoding on Linux. When .NET was created there was many character encodings used widely and UTF-8 was not dominant at the time - .NET has chose to use UTF-16. As UTF-8 became more popular it become more obvious that UTF-8 has many memory advantages over other encodings including UTF-16 and therefore its great adoption.
+
+## Benefits of using UTF-8 over UTF-16
```

Comment:
For completeness, we should also add a drawbacks section. The primary drawbacks are:

* If you perform heavy text processing or utilize globalization-related features heavily (e.g., a GUI application localized to your native language), UTF-16 is often faster than UTF-8 due to technical details of its implementation.
* For string which consist primarily of CJK characters, UTF-8 is less memory-efficient than UTF-16.

These conditions aren't expected to occur frequently in web applications or other application types where TTFB and memory utilization are critical metrics.